### ISSUE TITLE
Provide Kotlin friendly extensions

### ISSUE THREAD

**Issue by arriolac** [googlemaps/android-maps-utils#618]
**Is your feature request related to a problem? Please describe.**
This library should have Kotlin friendly extensions so that modern language features offered by Kotlin (e.g. extensions, lambdas, coroutines, etc.) can be used while interacting with this library. 

 **Describe the solution you'd like**
This feature should allow me to write idiomatic Kotlin. For example:

* I should be able to use Kotlin function extensions instead of using `*Util` classes that have static methods. This helps me discover functionality that the library offers.

_Before_:
```java
Polygon polygon = // some polygon
LatLng latlng = // some latlng
boolean result = PolygonUtil.containsLocation(latlng, polygon.getPoints(), true);
```

_After_:
```kotlin
val polygon: Polygon = // some polygon
val latlng: LatLng = // some latlng
val result: Boolean = polygon.contains(latLng)
```

* I should also be able to use named parameters with default values for methods that have several arguments. This helps me understand with code readability as I can see what dependencies I need to pass and which have default values set.

_Before_:
```java
GeoJsonLayer layer = new GeoJsonLayer(map, geoJsonFile, null, polygonManager, null, groundOverlayManager);
```

_After_:
```kotlin
val layer = GeoJsonLayer(
    map = map,
    geoJsonFile = geoJsonFile,
    polygonManager = polygonManager,
    groundOverlayManager = groundOverlayManager
)
```

etc.

 **Describe alternatives you've considered**
[Android KTX](https://developer.android.com/kotlin/ktx) in Android Jetpack is a good example of this.

 **Additional context**
This is in active development in the [android-maps-utils-ktx](https://github.com/googlemaps/android-maps-utils/tree/android-maps-utils-ktx) branch.

**Comment by arriolac** [comment]
Moved development to a new repo: https://github.com/googlemaps/android-maps-utils-ktx

**Comment by arriolac** [comment]
Kotlin extensions for this library have been released (version 0.1). Refer to https://github.com/googlemaps/android-maps-ktx to learn more.
